### PR TITLE
Firma 261 sign with fi re

### PR DESCRIPTION
**Pull request by SergioPH**

### CODE REVIEW

File: src/core/business/RequestSignerController.m [hunk 1]
```diff
@@ -63,6 +65,62 @@ - (void)sendNextRequest {
...
+	NSData *data = [PreSignXMLController buildDataForSigningPrechargedRequestInFIRe];
+	_wsController.delegate = self;
+	[_wsController postSignRequestWithFIRe:data code:code success:^(NSDictionary *content) {
+		NSDictionary *responseDict = [content objectForKey:kCfsigTag];
+		if ([[responseDict objectForKey:kOk] isEqualToString:kTrue]) {
+			[[self delegate] didReceiveCorrectSignResponseFromFIRe];
+		} else if ([[responseDict objectForKey:kOk] isEqualToString:kFalse]) {
+			[[self delegate] didReceiveErrorSignResponseFromFIRe:[responseDict objectForKey: kErrorFIReKey]];
+		} else {
+			[[self delegate] didReceiveErrorInPrechargedFIReRequest:NSLocalizedString(@"FIRe_error_in_server_message", nil)];
```
Comment (ggdiez) [inline src/core/business/RequestSignerController.m]: Could you use a localized extension?

File: src/ui/controllers/lists/UnassignedRequestTableViewController.m [hunk 2]
```diff
@@ -560,11 +588,141 @@ - (void)didReceiveRejectResult:(NSArray *)requestsSigned
     [self cancelEditing];
 }
 
+- (void)showFIRMeWebView:(NSURL *) url {
+	 dispatch_async(dispatch_get_main_queue(), ^{
+		 [SVProgressHUD dismiss];
+		 [self enableUserInteraction: true];
+		 [self.navigationController setNavigationBarHidden:YES animated:YES];
+		 [self.navigationController setToolbarHidden:YES animated:NO];
+		 self.webView = [[UIWebView alloc] initWithFrame: CGRectMake(0, 0, self.view.frame.size.width, self.view.frame.size.height)];
```
Comment (ggdiez) [inline src/ui/controllers/lists/UnassignedRequestTableViewController.m]: UIWebView is deprecated and it cause problems with App Store Connect. Can you use WKWebview? 
https://developer.apple.com/documentation/uikit/uiwebview

File: src/ui/controllers/lists/UnassignedRequestTableViewController.m [hunk 3]
```diff
@@ -560,11 +588,141 @@ - (void)didReceiveRejectResult:(NSArray *)requestsSigned
...
+		 [self.webView loadRequest: nsrequest];
+		 [self.view addSubview: self.webView];
+	 });
+}
+
+-(void) didReceiveCorrectSignResponseFromFIRe {
+	dispatch_async(dispatch_get_main_queue(), ^{
+		[SVProgressHUD dismissWithCompletion:^{
+			UIAlertController *alertController = [UIAlertController alertControllerWithTitle:NSLocalizedString(@"Info", nil) message:NSLocalizedString(@"Alert_View_Everything_Signed_Correctly", nil) preferredStyle:UIAlertControllerStyleAlert];
+			UIAlertAction *cancel = [UIAlertAction actionWithTitle:NSLocalizedString(@"Ok", nil) style:UIAlertActionStyleCancel handler:nil];
```
Comment (ggdiez) [inline src/ui/controllers/lists/UnassignedRequestTableViewController.m]: Review NSLocalizedString

File: src/ui/controllers/settings/SettingsVC.h [hunk 4]
```diff
@@ -8,7 +8,9 @@
 
 #import <UIKit/UIKit.h>
 #import "ServerListTVC.h"
+#import "SettingsCell.h"
+#import <WebKit/WebKit.h>
 
-@interface SettingsVC : PFBaseTVC <ServerListTVCDelegate>
+@interface SettingsVC : PFBaseTVC <ServerListTVCDelegate, SettingsCellDelegate, UIWebViewDelegate>
```
Comment (ggdiez) [inline src/ui/controllers/settings/SettingsVC.h]: Can you use WKWebView? 

https://developer.apple.com/documentation/uikit/uiwebview

File: src/ui/controllers/settings/SettingsVC.m [hunk 5]
```diff
@@ -129,52 +148,73 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 }
 
 #pragma mark - Navigation Methods
-    
+
+- (void) showLoginWebView {
+	dispatch_async(dispatch_get_main_queue(), ^{
+		self.webView = [[UIWebView alloc] initWithFrame: self.view.bounds];
```
Comment (ggdiez) [inline src/ui/controllers/settings/SettingsVC.m]: Can you user WKWebView?

Comment (SergioPH) [reply src/ui/controllers/settings/SettingsVC.m]: Done

File: src/ui/controllers/settings/SettingsVC.h (hunk 4 above)
Comment (SergioPH) [reply src/ui/controllers/settings/SettingsVC.h]: Done

File: src/ui/controllers/lists/UnassignedRequestTableViewController.m (hunk 2 above)
Comment (SergioPH) [reply src/ui/controllers/lists/UnassignedRequestTableViewController.m]: Done

File: src/ui/controllers/lists/UnassignedRequestTableViewController.m (hunk 3 above)
Comment (SergioPH) [reply src/ui/controllers/lists/UnassignedRequestTableViewController.m]: Done in all project

File: src/core/business/RequestSignerController.m (hunk 1 above)
Comment (SergioPH) [reply src/core/business/RequestSignerController.m]: Done

**Pull request merged**